kubectl-plugin/src/resources/volume.rs

```diff
@@ -19,7 +19,17 @@ impl CreateRows for Vec<openapi::models::Volume> {
     fn create_rows(&self) -> Vec<Row> {
         let mut rows: Vec<Row> = Vec::new();
         for volume in self {
-            let state = volume.state.as_ref().unwrap();
+            let state = volume
+                .state
+                .clone()
+                // If the state comes as empty fill in the spec data and mark the status as Unknown.
+                .unwrap_or(openapi::models::VolumeState {
```

Comment:
@paulyoong as it stands looks like we always have a state for a volume too, maybe we should change the openapi spec to reflect this?

Reply:
If I remember correctly I think we copy things from the spec into the state if we don't have one and then return this.